.github/workflows/deploy.yml

```diff
@@ -17,44 +17,33 @@ jobs:
         uses: docker/setup-qemu-action@v3
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
-      - name: Docker meta
-        id: meta
-        uses: docker/metadata-action@v5
-        with:
-          images: ${{ secrets.DOCKERHUB_USERNAME }}/${{ secrets.DOCKERHUB_RPC_REPO }}
-          tags: |
-            type=ref,event=tag
-            type=ref,event=pr
-            type=raw,value=latest,enable={{is_default_branch}}
-          labels: |
-            maintainer=bitcoindevproject
-            org.opencontainers.image.title=warnet-rpc
-            org.opencontainers.image.description=Warnet RPC server
-      - name: Login to Docker Hub
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Build and push production RPC image
+      - name: Build production RPC image
         uses: docker/build-push-action@v5
         with:
           file: resources/images/rpc/Dockerfile_prod
           platforms: linux/amd64,linux/arm64
-          context: src/warnet/templates/rpc
-          push: true
-          tags: ${{ steps.meta.outputs.tags }}
+          push: false
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ secrets.DOCKERHUB_RPC_REPO }}:latest
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
-      - name: Build and push dev RPC image
-        if: github.ref == 'refs/heads/main'
+  build-dev:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.conclusion == 'success'
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Build dev RPC image
         uses: docker/build-push-action@v5
         with:
           file: resources/images/rpc/Dockerfile_dev
           platforms: linux/amd64,linux/arm64
           context: src/warnet/templates/rpc
-          push: true
+          push: false
```

Comment:
7257b9c9ccb7d82a00639de2a6a364f30eac61ee

Is the title of this commit missing the word "don't" ?